scalatags/test/src/scalatags/generic/StyleSheetTests.scala

```diff
@@ -86,7 +86,7 @@ abstract class StyleSheetTests[Builder, Output <: FragT, FragT]
 
   def check(txt: String, expected: String) = {
     // augmentString = work around scala/bug#11125 on JDK 11
```

Comment:
If you use `linesIterator` then you can probably remove the `Predef.augmentSring` workaround and the comment.
Anyways, I would do that in a separate PR.

Reply:
Probably. I only replaced `lines` with `linesIterator` since it gave warnings about `lines` being deprecated or something.